enhancements/declarative-index-config.md

```diff
@@ -451,8 +451,8 @@ $ git status
 $ cd ..
 $ opm index validate community-operators
 marshal error: etcd.defaultchannel: cannot convert incomplete value "string" to JSON
-$ opm index update community-operator --tag=docker.io/mynamespace/community-operators:latest
-$ opm index validate docker.io/mynamespace/community-operators:latest 
+$ opm index build --configs=community-operator --tag=docker.io/my-namespace/community-operators:latest
+$ opm index validate --image=docker.io/my-namespace/community-operators:latest 
```

Comment:
I'm not sold on validate being an opm command, but if it exists, IMO the image should be taken in the same operand as the declarative directory would be; e.g. `opm index validate docker:docker.io/my-namespace/community-operators:latest` and `opm index validate ./community-operators`.